README.md

```diff
@@ -53,6 +53,9 @@ generic with minimal need for project-specific behavior.
 
 - AWS credentials configured in `~/.aws/credentials` and/or `~/.aws/config`
 
+- Users of macOS 11 (Big Sur) or later should follow additional steps mentioned 
+  in [Troubleshooting](#installing-python-383-on-macos-bigsur)
```

Comment:
Anchor is undefined. Link does not work.

Reply:
Don't know how I glossed over that mention of `bigsur`. It has been changed.